controllers/helpers.go

```diff
@@ -188,6 +188,7 @@ func GetCloudProviderSecret(d azure.ClusterScoper, namespace, name string, owner
 	secret.Data = map[string][]byte{
 		"control-plane-azure.json": controlPlaneData,
 		"worker-node-azure.json":   workerNodeData,
+		"azure.json":               controlPlaneData,
```

Comment:
/hold

can you please add a code comment that says this is included for backward compatibility?